tool/site/client/static/styles.css

```diff
@@ -96,4 +96,65 @@ border-top-color:#fff
 }
 .nhsuk-loader--white .nhsuk-loader__text {
 color:#fff
-}
\ No newline at end of file
+}
+
+.pagination-container {
+	display: flex;
+	list-style-type: none;
+	justify-content: center;
+}
+.pagination-container .pagination-item {
+	padding: 0 12px;
+	height: 32px;
+	text-align: center;
+	margin: auto 4px;
+	color: rgba(0, 0, 0, 0.87);
```

Comment:
not super important but we could probably use css variables here for color values

Reply:
will update in future pr